### PR TITLE
add unit test and types

### PR DESCRIPTION
**Pull request by Jimexist**
```release-note
add unit test and types
```

/cc @kdada
/cc @ddysher 
/cc @zoumo

### CODE REVIEW

**Comment by ddysher** [comment]
Still need to 
- add OWNERS
- update labels
- add LICENSE and header for each file

File: cmd/rooms/main.go [hunk 1]
```diff
@@ -1,5 +1,29 @@
 package main
 
-func main() {
+import (
+	"github.com/caicloud/nirvana/cli"
+	"github.com/caicloud/nirvana/log"
+	"github.com/caicloud/nirvana_example_meeting_rooms/cmd/rooms/server"
+	"github.com/caicloud/nirvana_example_meeting_rooms/pkg/db"
+	"github.com/spf13/cobra"
```
Comment (ddysher) [inline cmd/rooms/main.go]: we should require this... need to fix in nirvana? @zoumo

File: cmd/schedule/main.go [hunk 2]
```diff
@@ -1,5 +1,29 @@
 package main
 
-func main() {
+import (
+	"github.com/caicloud/nirvana/cli"
+	"github.com/caicloud/nirvana/log"
+	"github.com/caicloud/nirvana_example_meeting_rooms/cmd/rooms/server"
```
Comment (ddysher) [inline cmd/schedule/main.go]: this import is wrong?

**Comment by Jimexist** [comment]
https://github.com/caicloud/nirvana_example_meeting_rooms/blob/master/OWNERS @ddysher

**Comment by Jimexist** [comment]
@ddysher what did you mean by _update labels_? Do i need to manually update the labels? Can it be automated by the bot?

**Comment by Jimexist** [comment]
@ddysher I want this repo to be iterated fastly without being blocked by reviewing process which as of now is quite slow. I think once it has reached a maturing point we can do a code auditing on all of the example code as well as docs in one pass.

File: cmd/schedule/main.go (hunk 2 above)
Comment (Jimexist) [reply cmd/schedule/main.go]: not really

File: cmd/rooms/main.go (hunk 1 above)
Comment (Jimexist) [reply cmd/rooms/main.go]: tracked with #4

**Pull request merged**